### DISCUSSION TITLE
Kill a conversation session

### DISCUSSION THREAD

**Discussion by Aochong-Li** [llm-workflow-engine/llm-workflow-engine#114] [Q&A]
Hi,
After using chatgpt.ask(prompt), how can I kill the conversation and start a new one, so that there won't be too many session open on the browser? I tried chatgpt._cleanup() and _cleanup_divs(), but they don't seem to work. Thanks a lot lol.

**Comment by thehunmonkgroup** [comment]
It already re-uses the existing browser session, you do not need to do this kind of cleanup.
